src/main/java/org/folio/marccat/resources/TenantAPI.java

```diff
@@ -25,29 +33,26 @@
   private TenantRefService tenantRefService;
 
   @PostMapping
-  public ResponseEntity<String> create(
-    @RequestHeader(Global.OKAPI_TENANT_HEADER_NAME) String tenant,
-    @RequestHeader(Global.OKAPI_URL) String okapiUrl,
-    @RequestHeader(Global.OKAPI_TO_URL) String okapiUrlTo,
-    @RequestBody TenantAttributes attributes
-  ) throws SQLException, IOException {
+  public ResponseEntity<String> create(@RequestHeader(Global.OKAPI_TENANT_HEADER_NAME) String tenant,
+      @RequestHeader(Global.OKAPI_URL) String okapiUrl, @RequestHeader(Global.OKAPI_TO_URL) String okapiUrlTo,
+      @RequestBody TenantAttributes attributes) throws SQLException, IOException {
     addHeaders(tenant, okapiUrl, okapiUrlTo);
     tenantService.createTenant(tenant, okapiUrl);
-    if(!okapiUrl.isEmpty())
+    if (!okapiUrl.isEmpty())
       tenantRefService.loadData(attributes, okapiHeaders);
     return new ResponseEntity("Success", CREATED);
   }
 
-  public void addHeaders(@RequestHeader(Global.OKAPI_TENANT_HEADER_NAME) String tenant, @RequestHeader(Global.OKAPI_URL) String okapiUrl, @RequestHeader(Global.OKAPI_TO_URL) String okapiUrlTo) {
+  public void addHeaders(@RequestHeader(Global.OKAPI_TENANT_HEADER_NAME) String tenant,
+      @RequestHeader(Global.OKAPI_URL) String okapiUrl, @RequestHeader(Global.OKAPI_TO_URL) String okapiUrlTo) {
```

Comment:
Why are these strings annotated with `@RequestHeader`? Does it do anything in a non-annotated spring method? Shouldn't the injection happen in `create`, which is mapped to the "POST" method? Is this method used outside this class? Could it be private?

Reply:
I think @carmentrazza will be able to answer your question better

Reply:
Yes, it is a private method and must be without @RequestHeader

Reply:
private void addHeaders(String tenant, String okapiUrl, String okapiUrlTo) {
    okapiHeaders.put(Global.OKAPI_TENANT_HEADER_NAME, tenant);
    okapiHeaders.put(Global.OKAPI_URL, okapiUrl);
    okapiHeaders.put(Global.OKAPI_TO_URL, okapiUrlTo);
  }

Reply:
You can change this method

Reply:
I have changed it